chain/chain/src/runtime/tests.rs

```diff
@@ -78,8 +78,13 @@ impl NightshadeRuntime {
         gas_limit: Gas,
         challenges_result: &ChallengesResult,
     ) -> (StateRoot, Vec<ValidatorStake>, Vec<Receipt>) {
-        // TODO(congestion_control)
-        let congestion_info_map = HashMap::new();
+        // TODO(congestion_control): pass down prev block info and read congestion info from there
+        // For now, just use default.
+        let epoch_id =
+            self.epoch_manager.get_epoch_id_from_prev_block(block_hash).unwrap_or_default();
+        let shard_ids = self.epoch_manager.shard_ids(&epoch_id).unwrap();
+        let congestion_info_map: HashMap<ShardId, CongestionInfo> =
+            shard_ids.into_iter().map(|shard_id| (shard_id, CongestionInfo::default())).collect();
         let mut result = self
```

Comment:
@wacban can you please take a look a this change to make test green? I know it's not the perfect solution, yet, but I I feel like the PR is already big an complicated, so I chose the easiest way to make the test green. But maybe it should still be somewhat correct.

Reply:
lgtm

Reply:
I'm fine with anything as long as there is a todo 😉